spk/tvheadend/src/service-setup.sh

```diff
@@ -72,13 +66,13 @@ service_preupgrade ()
         done
         # Move recording directories from package root into var
         UPGRADE_CFG_DIR="${SYNOPKG_PKGDEST}/var/dvr/config"
```

Comment:
Same here, `${SYNOPKG_PKGVAR}` can probably be used.
Also, same variable is also used below so probably best to move outside of function for using everywhere.